maps/src/MapMockObserver.cxx

```diff
@@ -0,0 +1,171 @@
+#include <pybindings.h>
+#ifdef OPENMP_FOUND
+#include <omp.h>
+#endif
+
+#include <G3Module.h>
+#include <G3Timestream.h>
+#include <G3Quat.h>
+#include <G3Data.h>
+#include <G3Map.h>
+#include <maps/G3SkyMap.h>
+#include <maps/pointing.h>
+#include <calibration/BoloProperties.h>
+
+class MapMockObserver : public G3Module {
+public:
+	MapMockObserver(std::string pointing, std::string timestreams,
+	    double band, G3SkyMapConstPtr T, G3SkyMapConstPtr Q,
+	    G3SkyMapConstPtr U, std::string bolo_properties_name);
+	virtual ~MapMockObserver() {}
+
+	void Process(G3FramePtr frame, std::deque<G3FramePtr> &out);
+
+private:
+	void BinTimestream(const G3Timestream &det, double weight,
```

Comment:
I think this `BinTimestream` method is a copy/paste snafu?

Reply:
Indeed. Nice catch! Deleted.